### ISSUE TITLE
`DisclosureTriangleRole` model wrong?

### ISSUE THREAD

**Issue by dummdidumm** [A11yance/axobject-query#319]
In #273 the `DisclosureTriangleRole` model was adjusted and constraints/attributes were added. If I'm understanding this correctly, that means that a `<summary>` tag now either needs a `open` or `aria-expanded` attribute, but both these AFAIK should be on the parent `<details>` tag instead. So if I'm understanding this correctly, the `DisclosureTriangleRole` model should instead be reduced to

```js
const DisclosureTriangleRole: AXObjectModelDefinition = {
  relatedConcepts: [
    {
      module: 'HTML',
      concept: {
        constraints: [
          'scoped to a details element',
        ],
        name: 'summary',
      },
    },
  ],
  type: 'widget',
};
```

It could also very well be that I'm understanding the semantic meaning of `attributes` incorrectly, in which case could you briefly explain what it means?

**Comment by jessebeach** [comment]
Here is the `summary` html element spec for future reference: https://html.spec.whatwg.org/multipage/interactive-elements.html#the-summary-element

**Comment by jessebeach** [comment]
It looks like the `summary` element is mapped to the `DisclosureTriangle` concept. 

<img width="648" alt="Screenshot 2023-08-12 at 9 27 21 PM" src="https://github.com/A11yance/axobject-query/assets/345913/306ca16e-0cb2-4b23-80df-180ad77cad8e">

**Comment by jessebeach** [comment]
@dummdidumm your proposal is correct.

**Comment by jessebeach** [comment]
Fixed in #339

**Comment by dummdidumm** [comment]
@jessebeach sorry to ping - when can we expect this to get released?

**Comment by jessebeach** [comment]
I'll try to get a release out on Sunday.

**Comment by jessebeach** [comment]
@dummdidumm I just published v4.0.0: https://www.npmjs.com/package/axobject-query
